packages/datatrak-web/src/components/Button.tsx

```diff
@@ -7,7 +7,9 @@ import { To, Link as RouterLink } from 'react-router-dom';
 import { Tooltip, Button as UIButton } from '@tupaia/ui-components';
 import styled from 'styled-components';
 
-const StyledButton = styled(UIButton)`
+const StyledButton = styled(UIButton).attrs({
+  style: { verticalAlign: 'baseline' }, // Override MUI-generated styling in <style> tag
```

Comment:
@acdunham Do you know of a more “correct” way to override the `<style>` tag that MUI generates?

I assume there must be some way that tells MUI to generate its `<style>` with this style baked in, but not sure what to Google

Reply:
I'm not sure tbh. You could try setting in the overrides in the `theme.ts` file. but I'm not sure if it would override these inline styles. Otherwise you could find another way to achieve the same styling, e.g with flexbox.

Reply:
`theme.ts` was exactly it 🙏